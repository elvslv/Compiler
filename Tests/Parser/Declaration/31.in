type TRecord = record a: array[1..10] of array[1..10] of integer; end;