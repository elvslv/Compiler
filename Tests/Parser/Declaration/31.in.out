type TRECORD = record 
    var A : array [const 1..const 10] of array [const 1..const 10] of INTEGER;
end;
